src/wallet/ismine.h

```diff
@@ -28,8 +28,8 @@ enum isminetype
 /** used for bitflags of isminetype */
 typedef uint8_t isminefilter;
 
-isminetype IsMine(const CKeyStore& keystore, const CScript& scriptPubKey);
-isminetype IsMine(const CKeyStore& keystore, const CTxDestination& dest);
+isminetype IsMine(const CWallet& wallet, const CScript& scriptPubKey);
```

Comment:
It's a shame that the declaration and definition have different argument names (`wallet` here in the decl and `keystore` in the def)